### ISSUE TITLE
the path to forever?

### ISSUE THREAD

**Issue by TinajaLabs** [reederz/puppet-nodered#5]
Had to change the path to forever for my install:

from: `/usr/local/bin/forever`
to: `/usr/lib/node_modules/forever/bin/forever`

This is in `/node-red/manifests/init.pp`

```
  exec {  'run_nodered_forever':
    # command  =>  "(/usr/local/bin/forever stop nodered || echo 'Node-RED was not running') && /usr/local/bin/forever start --uid 'nodered' --append ${install_dir}/red.js --settings /etc/nodered/settings.js -v",
    command  =>  "(/usr/lib/node_modules/forever/bin/forever stop nodered || echo 'Node-RED was not running') && /usr/lib/node_modules/forever/bin/forever start --uid 'nodered' --append ${install_dir}/red.js --settings /etc/nodered/settings.js -v",
    provider => 'shell',
    require  =>  Class['nodered::configure'],
  }
```

```
  cron {  'start_nodered_on_reboot':
    # command =>  "(/usr/local/bin/forever stop nodered || echo 'Node-RED was not running') && /usr/local/bin/forever start --uid 'nodered' --append ${install_dir}/red.js --settings /etc/nodered/settings.js -v",
    command =>  "(/usr/lib/node_modules/forever/bin/forever stop nodered || echo 'Node-RED was not running') && /usr/lib/node_modules/forever/bin/forever start --uid 'nodered' --append ${install_dir}/red.js --settings /etc/nodered/settings.js -v",
    special =>  'reboot',
    require =>  Class['nodered::configure'],
  }
```

The error I was getting:

```
Error: /bin/sh: 1: /usr/local/bin/forever: not found
Node-RED was not running
/bin/sh: 1: /usr/local/bin/forever: not found
```

**Comment by reederz** [comment]
Thanks! What's your operating system?

**Comment by TinajaLabs** [comment]
Target is Ubuntu 14.04 running on an Odroid C1, arm based.  I'm basically loading up a Raspi like computer with node-red, graphite/grafana, rabbitMQ as a home automation hub.

Puppet Master is running on Centos 6.5 running in an Ubuntu KVM.

Thanks for this module.

**Comment by reederz** [comment]
You're welcome. Well, I've developed this module for Ubuntu 14.04 so it's weird that path to _forever_ is different on your instance. Maybe arm based version is different. Regardless, I think I can dinamically determine the path instead of hardcoding it. I'll fix it soon.

**Comment by TinajaLabs** [comment]
In case it helps, here are more specifics on the distro I am using:

Here's the page where I DLd the distro: 
http://east.us.odroid.in/ubuntu_14.04lts/

I DLd this version for the C1:
http://east.us.odroid.in/ubuntu_14.04lts/ubuntu-14.04.3lts-lubuntu-odroid-c1-20150811.img.xz

**Comment by reederz** [comment]
Sorry for such a long delay. Your issue is solved with release `0.2.0` .

**Comment by TinajaLabs** [comment]
Thank you.
